docs/getting-started-guides/kops.md

```diff
@@ -0,0 +1,158 @@
+for `dev`), or at your domain registrar if it is a root domain name (e.g. `example.com` would need to be configured
+where you bought `example.com`).
+
+This step is easy to mess up (it is the #1 cause of problems!)  You can double-check that
+your cluster is configured correctly if you have the dig tool by running:
+
+`dig NS dev.example.com`
+
+You should see the 4 NS records that Route53 assigned your hosted zone.
+
+### (3/5) Create an S3 bucket to store your clusters state
+
+kops lets you manage your clusters even after installation.  To do this, it must keep track of the clusters
+that you have created, along with their configuration, the keys they are using etc.  This information is stored
+in an S3 bucket.  S3 permissions are used to control access to the bucket.
+
+Multiple clusters can use the same S3 bucket, and you can share an S3 bucket between your colleagues that
+administer the same clusters - this is much easier than passing around kubecfg files.  But anyone with access
+to the S3 bucket will have administrative access to all your clusters, so you don't want to share it beyond
+the operations team.
+
+So typically you have one S3 bucket for each ops team (and often the name will correspond
+to the name of the hosted zone above!)
+
+In our example, we chose `dev.example.com` as our hosted zone, so let's pick `clusters.dev.example.com` as
+the S3 bucket name.
+
+* Export `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+
+* Create the S3 bucket using `aws s3 mb s3://clusters.dev.example.com`
+
+* You can `export KOPS_STATE_STORE=s3://clusters.dev.example.com` and then kops will use this location by default. 
+   We suggest putting this in your bash profile or similar. 
+
+
+### (4/5) Build your cluster configuration
+
+Run "kops create cluster" to create your cluster configuration:
+
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
+
+kops will create the configuration for your cluster.  Note that it _only_ creates the configuration, it does
+not actually create the cloud resources - you'll do that in the next step with a `kops update cluster`.  This
+give you an opportunity to review the configuration or change it.
+
+It prints commands you can use to explore further:
+
+* List your clusters with: `kops get cluster`
+* Edit this cluster with: `kops edit cluster useast1.dev.example.com`
+* Edit your node instance group: `kops edit ig --name=useast1.dev.example.com nodes`
+* Edit your master instance group: `kops edit ig --name=useast1.dev.example.com master-us-east-1c`
+
+If this is your first time using kops, do spend a few minutes to try those out!  An instance group is a
+set of instances, which will be registered as kubernetes nodes.  On AWS this is implemented via auto-scaling-groups.
+You can have several instance groups, for example if you wanted nodes that are a mix of spot and on-demand instances, or
+GPU and non-GPU instances.
```

Comment:
That's fair - I only mention it because of the suggestion that the reader spend some time playing with IGs, since the settings most people would play around with would involve cheap clusters to experiment. Leaving the details out here makes sense in general though of course.